site/src/load.rs

```diff
@@ -64,23 +64,32 @@ impl TryCommit {
     }
 }
 
+/// Keys for accessing various services
+///
+/// At the moment only used for accessing GitHub
 #[derive(Debug, Default, Deserialize)]
 pub struct Keys {
+    /// GitHub API token from the `GITHUB_API_TOKEN` env variable
     pub github: Option<String>,
```

Comment:
These fields might have better names like `github_api_token` and `github_webhook_secret`.